pkg/apiserver/registry/inventory/virtualmachine/rest.go

```diff
@@ -83,6 +84,37 @@ func (r *REST) ShortNames() []string {
 	return []string{"vm"}
 }
 
+func (r *REST) Update(ctx context.Context, name string, objInfo rest.UpdatedObjectInfo, _ rest.ValidateObjectFunc,
```

Comment:
nit: Add a comment saying only spec fields can be updated via the rest interface.

Reply:
its kind of expected.. Status is runtime, and Spec is desired.. So not required.